metacat-main/src/main/java/com/netflix/metacat/main/services/notifications/sns/SNSNotificationServiceImpl.java

```diff
@@ -65,33 +61,35 @@
  */
 @Slf4j
 public class SNSNotificationServiceImpl implements NotificationService {
-    private static final String EXCEED_PARTITION_ID_ALLOWED_MAX_NUMBER = "Exceed partitionId allowed max number ";
-    private static final String PARTITION_ID_UNENABLED = "PartitionId attachment not enabled";
-    private static final String PARTITION_ID_ATTACHED = "PartitionId attached ";
+    private static final String PARTITION_ID_UNENABLED = "Latest PartitionId attachment not enabled";
```

Comment:
We should use Enum in this case and pass the string to the event.

Reply:
sure.